cookbooks/kibana/attributes/default.rb

```diff
@@ -1,4 +1,5 @@
 # Attributes for Kibana
 
-default["kibana"]["dir"]   = "/home/#{node["delk_user"]}/kibana"
-default["kibana"]["port"]  = "5601"
+default["kibana"]["dir"]      = "/home/#{node["delk_user"]}/kibana"
+default["kibana"]["port"]     = "5601"
+default["kibana"]["all_dirs"] = %w( . conf )
```

Comment:
I did this to reduce duplication of code, I create the parent directory only, the rest is created using that iteration, and then I set permissions and mode for both the parent directory and children.